columnflow/plotting/plot_util.py

```diff
@@ -17,6 +17,51 @@
 od = maybe_import("order")
 
 
+def apply_variable_settings(hists: dict, variable_settings: dict | None = None) -> dict:
+    # check if there are variable settings to apply
+    if not variable_settings:
+        return hists
+    relevant_variables = set(list(hists.values())[0].axes.name) & set(variable_settings.keys())
+    if not relevant_variables:
+        return hists
+
+    # apply all settings
+    for var in relevant_variables:
+        var_settings = variable_settings[var]
+        for key, h in hists.items():
+            # apply rebinning setting
+            rebin_factor = int(var_settings.get("rebin", 1))
```

Comment:
Just a question: can there be cases where `rebin` is present in `var_settings`, but `None`?

Reply:
The settings are parsed to either a bool, float or string, so None values should not be possible.